src/vm/jitinterface.cpp

```diff
@@ -1302,9 +1302,23 @@ struct TryResolveTokenFilterParam
 
 bool isValidTokenForTryResolveToken(CEEInfo* info, CORINFO_RESOLVED_TOKEN* resolvedToken)
 {
-    LIMITED_METHOD_CONTRACT;
+    CONTRACTL {
+        NOTHROW;
+        GC_TRIGGERS;
```

Comment:
I do not think that it is safe for the C++ exception filters to be `GC_TRIGGERS`. I understand that it was sort there before - just undetected, but there is a bunch more code running in the `EX_TRY/EX_CATCH` now.